utils/generateTitleTest.py

```
@@ -1,20 +1,47 @@
-    title = tokenizer.decode(summary_ids[0], skip_special_tokens=True)
-
-    print("제목:", title)
-    return title
-
+from transformers import PreTrainedTokenizerFast                  # 한국어 제목
+from transformers import BartForConditionalGeneration             # 한국어 제목
+from transformers import T5ForConditionalGeneration, T5Tokenizer  # 영어 제목
+
+
+def generate_title(text, lang):
+    if text == "error1" or "error2":  # 요약 실패
```

Ruff (0.8.2)

Use `True` instead of `... or True`

Replace with `True`

(SIM222)